geometry/segment2d.cpp

```diff
@@ -1,8 +1,10 @@
 #include "geometry/segment2d.hpp"
 
+#include "geometry/line2d.hpp"
```

Comment:
Now line2d.hpp and segment2d.hpp include each other.

Reply:
line2d.hpp includes, but segment2d.hpp not. segment2d.cpp does.